tox.ini

```diff
@@ -1,10 +1,20 @@
 [tox]
-envlist = {py34, py35, py36, py37}
+envlist = {py34, py35, py36, py37}-nameko{2.6, 2.7, 2.8, 2.9, 2.10, 2.11}
 skipsdist = True
 
 [testenv]
 whitelist_externals = make
 usedevelop = true
 extras = dev
+deps =
+    nameko{2.6,2.7}: pytest<3.3.0
+    nameko{2.6,2.7}: eventlet<0.22.0
```

Comment:
Can we combine both `pytest<3.3.0` and `eventlet<0.22.0` (to have more than one dependency) for Nameko 2.6 and 2.7 without duplicating the line with `nameko{2.6, 2.7}`?

Reply:
Unfortunately not, `tox` expects one dependency per line and doesn't seem to support applying the same factor condition for multiple dependencies.

Reply:
Oh, I see. It's a bit unfortunate. Thanks for trying!